### PR TITLE
ensures versions are retrieved properly

### LINKED ISSUE #109: All Release Reference Documentation Links are not working anymore
All [Release Reference Documentation Links](https://projectreactor.io/docs/core/release/reference/) are not working anymore

## Expected Behavior

Open the Release version reference documentation

## Actual Behavior

Receipt of a general 404 error page.

## Steps to Reproduce

Open the Release Reference documentation 
--> [CLICK ME](https://projectreactor.io/docs/core/release/reference/) <--

## Possible Solution

Use the [Snapshot documentation](https://projectreactor.io/docs/core/snapshot/reference/) until fixed.

### PR DESCRIPTION
**Pull request by OlegDokuka**
closes #109

### CODE REVIEW

File: src/main/java/io/projectreactor/ModuleUtils.java [hunk 1]
```diff
@@ -102,6 +103,54 @@ public static void loadModuleVersionsFromArtifactoryVersionsSearch(String json,
 		versions.forEach(v -> tryAddVersion(module, v));
 	}
 
+	public static void fetchVersionsFromSonotype(Map<String, Module> modules, String... moduleNames) {
```
Comment (SpikeBlues) [inline src/main/java/io/projectreactor/ModuleUtils.java:106]: I guess there's a typo with the method name. Should it be `sonatype` instead?
```suggestion
	public static void fetchVersionsFromSonatype(Map<String, Module> modules, String... module names) {
```

File: src/main/java/io/projectreactor/Application.java [hunk 2]
```diff
@@ -94,6 +94,10 @@ public final class Application {
 		ModuleUtils.fetchVersionsFromArtifactory(modules, "core", "test", "adapter",
 				"extra", "netty", "nettyArchive", "kafka", "rabbitmq", "BlockHound",
 				"kotlin", "pool");
+		//then get the versions from Sonotype
```
Comment (violetagg) [inline src/main/java/io/projectreactor/Application.java:97]:
```suggestion
		//then get the versions from Sonatype
```

File: src/main/java/io/projectreactor/ModuleUtils.java [hunk 3]
```diff
@@ -102,6 +103,54 @@ public static void loadModuleVersionsFromArtifactoryVersionsSearch(String json,
...
+					                               .doOnNext(errorBody -> LOGGER.warn("Couldn't scrape versions for {}: {} - {}",
+							                               module.getName(), r.status(), errorBody));
+				                 }
+			                 })
+			                 .then(Mono.fromCallable(module::sortAndDeduplicateVersions));
+		    })
+		    .blockLast();
+	}
+
+	public static void loadModuleVersionsFromSonotypeVersionsSearch(String json, Module module) {
```
Comment (violetagg) [inline src/main/java/io/projectreactor/ModuleUtils.java:138]:
```suggestion
	public static void loadModuleVersionsFromSonatypeVersionsSearch(String json, Module module) {
```

**Review by violetagg — CHANGES_REQUESTED** [review]

File: src/main/java/io/projectreactor/ModuleUtils.java [hunk 4]
```diff
@@ -102,6 +103,54 @@ public static void loadModuleVersionsFromArtifactoryVersionsSearch(String json,
 		versions.forEach(v -> tryAddVersion(module, v));
 	}
 
+	public static void fetchVersionsFromSonatype(Map<String, Module> modules, String... module names) {
```
Comment (violetagg) [inline src/main/java/io/projectreactor/ModuleUtils.java:106]:
```suggestion
	public static void fetchVersionsFromSonatype(Map<String, Module> modules, String... moduleNames) {
```

**Review by violetagg — APPROVED** [review]

**Pull request merged**